nearcore/src/config.rs

```diff
@@ -599,6 +599,15 @@ impl Config {
             &self.tracked_accounts,
         )
     }
+
+    pub fn contract_cache_path(&self) -> Cow<'_, Path> {
```

Comment:
nit:  I don't think `cow` here is worth the complexity, can you just return (cloned) owned Path?